src/index.jsx

```diff
@@ -7,7 +7,7 @@ import { HelmetProvider } from 'react-helmet-async';
 
 import { SUPPORTED_LANGUAGES, DEFAULT_LANGUAGE } from './constants/constants';
 import store from 'state/store';
-import App from './App';
+import App from './App.jsx';
```

Comment:
mmm if you have to add .jsx extension in the import, maybe there is something wrong with the project config

Reply:
You are right. I've already deleted that extension and left just `./App`